### DISCUSSION TITLE
Is it possible to split all vitest tests between different parallel buildkite agents?

### DISCUSSION THREAD

**Discussion by gvocale** [vitest-dev/vitest#4613] [General]
My Vitest tests run in 15 min in Buildkite. They are currently the slowest process, since our e2e test, divided between 5 Buildkite parallel agent, take 7 min max.

Is it possible to split Vitest tests in similar fashion between different Buildkite agents? This way I could split them between two different agents / machines and complete they in 7~min ideally.

Thanks

**Comment by sheremet-va** [comment]
https://vitest.dev/guide/cli.html#shard
